SQL_files/baseball_batting_average.sql

```diff
@@ -24,7 +24,7 @@ Table: annual table (BA annually)
 */
 DROP TABLE IF EXISTS BATTERS_ANNUALLY;
 CREATE TABLE BATTERS_ANNUALLY AS
-(SELECT baseball.batter_counts.batter AS Batter, REPLACE(YEAR(baseball.game.local_date),',','') AS The_Year, 
```

Comment:
just removing the empty space you found for an old fire? or are you using the baseball stuff in this assignment?